libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/SettingsFragment.kt

```diff
@@ -165,37 +143,9 @@ class SettingsFragment : PreferenceFragmentCompat(), SharedPreferences.OnSharedP
             prefUI.summary = prefUI.entry
         if (prefUI is EditTextPreference)
             prefUI.summary = prefUI.text
+   }
 
-        if (CycleStreetsPreferences.PREF_MAPSTYLE_KEY == key)
-            setMapFileSummary((prefUI as ListPreference).value)
-    }
-
-    private fun setMapFileSummary(style: String) {
-        val pref = findPreference<Preference>(CycleStreetsPreferences.PREF_MAPFILE_KEY) ?: return
-        val mapfilePref = pref as ListPreference
-
-        val enabled = style == CycleStreetsPreferences.MAPSTYLE_MAPSFORGE
-        mapfilePref.isEnabled = enabled
-
-        if (!enabled)
-            return
-
-        if (mapfilePref.entryValues.isEmpty()) {
-            mapfilePref.isEnabled = false
-            MessageBox.YesNo(requireView(), R.string.settings_no_map_packs) { _, _ -> MapPack.searchGooglePlay(requireContext()) }
-            return
-        }
-
-        val mapfile = CycleStreetsPreferences.mapfile()
-        var index = mapfilePref.findIndexOfValue(mapfile)
-        if (index == -1)
-            index = 0 // default to something
-
-        mapfilePref.setValueIndex(index)
-        mapfilePref.summary = mapfilePref.entries[index]
-    }
-
-    private fun setAccountSummary() {
+   private fun setAccountSummary() {
```

Comment:
I think you've accidentally de-indented this by 1 character 😂 

I clearly don't care enough to request a change, but you may feel like fixing it as part of your broader PR.